stdlib/random.ml

```diff
@@ -93,6 +93,31 @@ module State = struct
     then invalid_arg "Random.int"
     else intaux s bound
 
+  let rec int63aux s n =
+    let max_int_32 = (1 lsl 30) + 0x3FFFFFFF in (* 0x7FFFFFFF *)
+    let b1 = bits s in
+    let b2 = bits s in
+    let (b, max_int) =
+      if n <= max_int_32 then
+        (* 32 random bits on both 64-bit OCaml and JavaScript *)
+        (((b2 lsl 30) lor b1) land max_int_32, max_int_32)
```

Comment:
@damiendoligez prefers the high bits to the low bits, so what about
```suggestion
        ((((b2 land 0x30000000) lsl 2) lor b1) land max_int_32, max_int_32)
```
Pushing his logic further, we should combine the 16 top bits from b1 and from b2, and something similar in the b3 case below, but maybe that's enough for now.

Reply:
@dra27 could  you please accept the suggestion or reject it explicitly?  We need to make progress with PRs.